smart-contract-extensions/test/hooks/discountCodeHook.js

```diff
@@ -214,7 +214,7 @@ contract('DiscountCodeHook', accounts => {
           from: keyBuyer,
         }
       ),
-      'ONLY_LOCK_MANAGER.'
+      "VM Exception while processing transaction: reverted with reason string 'ONLY_LOCK_MANAGER'"
```

Comment:
one thing I dont get here is why this was throwing, when others `reverts` test were not, such as 

https://github.com/unlock-protocol/unlock/blob/065eb9ef5488eecb30fc58c060e21d5af1d92b54/smart-contract-extensions/test/keyPurchaser.js#L44-L52

Reply:
Oh well.